test/load/allocation/autoscaler.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2020 Google LLC All Rights Reserved.
```

Comment:
This is a new file right? If so, can you set the copyright date to the current year?